static/js/components/email/EmailCompositionDialog_test.js

```diff
@@ -166,14 +166,22 @@ describe('EmailCompositionDialog', () => {
     renderDialog({
       filters: [{
         id: '1',
-        name: "key",
-        value: "test"
+        name: "program.enrollments.semester",
+        value: "2015"
+      }, {
+        id: '2',
+        name: "ES",
+        value: "foo"
       }]
     }, { renderRecipients: SEARCH_RESULT_EMAIL_CONFIG.renderRecipients });
 
     assert.include(
-      getDialog().querySelector('.sk-selected-filters-option__name').textContent,
-      "key: test"
+      getDialog().querySelector('.sk-selected-filters').textContent,
+      "Semester: 2015"
+    );
+    assert.include(
+      getDialog().querySelector('.sk-selected-filters').textContent,
+      "Spain: foo"
```

Comment:
This doesn't seem like a valid test case. A valid test case might be `profile.birth_country: ES` converted into `Country of Birth: Spain`. In fact, I just manually tested that, and it doesn't seem like your pull request correctly does that conversion.